integration-tests/kyuubi-kubernetes-it/src/test/scala/org/apache/kyuubi/kubernetes/test/spark/SparkOnKubernetesTestsSuite.scala

```diff
@@ -147,23 +147,13 @@ class KyuubiOperationKubernetesClusterClientModeSuite
       batchRequest.getConf.asScala.toMap,
       batchRequest)
 
-    eventually(timeout(3.minutes), interval(50.milliseconds)) {
+    // [KYUUBI #4467] KubernetesApplicationOperator doesn't support client mode
+    eventually(timeout(1.minutes), interval(50.milliseconds)) {
```

Comment:
As we discussed before, let's just ignore this test case, and restore it after fixing K8s client mode.

Reply:
Sure